corehq/apps/hqwebapp/views.py

```diff
@@ -705,13 +705,135 @@ def jserror(request):
+        'datetime': datetime.utcnow(),
+        'self_started': '<unknown>',
+        'has_handoff_info': '<unknown>',
+        'project_description': '<unknown>',
+        'sentry_error': '{}{}'.format(getattr(settings, 'SENTRY_QUERY_URL', ''), report['sentry_id'])
+    }
+    if domain_object:
+        current_project_description = domain_object.project_description if domain_object else None
+        new_project_description = post_params.get('project_description')
+        if (domain_object and couch_user.is_domain_admin(domain=domain) and new_project_description
+                and current_project_description != new_project_description):
+            domain_object.project_description = new_project_description
+            domain_object.save()
+
+        message += ((
+            "software plan: {software_plan}\n"
+        ).format(
+            software_plan=Subscription.get_subscribed_plan_by_domain(domain),
+        ))
+
+        debug_context.update({
+            'self_started': domain_object.internal.self_started,
+            'has_handoff_info': bool(domain_object.internal.partner_contact),
+            'project_description': domain_object.project_description,
+        })
+
+    subject = '{subject} ({domain})'.format(subject=report['subject'], domain=domain)
+
+    if full_name and not any([c in full_name for c in '<>"']):
+        reply_to = '"{full_name}" <{email}>'.format(**report)
+    else:
+        reply_to = report['email']
+
+    # if the person looks like a commcare user, fogbugz can't reply
+    # to their email, so just use the default
+    if settings.HQ_ACCOUNT_ROOT in reply_to:
+        reply_to = settings.SERVER_EMAIL
+
+    message += "Message:\n\n{message}\n".format(message=report['message'])
+    if post_params.get('five-hundred-report'):
+        extra_message = ("This message was reported from a 500 error page! "
+                         "Please fix this ASAP (as if you wouldn't anyway)...")
+        extra_debug_info = (
+            "datetime: {datetime}\n"
+            "Is self start: {self_started}\n"
+            "Has Support Hand-off Info: {has_handoff_info}\n"
+            "Project description: {project_description}\n"
+            "Sentry Error: {sentry_error}\n"
+        ).format(**debug_context)
+        traceback_info = cache.cache.get(report['500traceback']) or 'No traceback info available'
+        cache.cache.delete(report['500traceback'])
+        message = "\n\n".join([message, extra_debug_info, extra_message, traceback_info])
+
+    email = EmailMessage(
+        subject=subject,
+        body=message,
+        to=[to_email],
+        headers={'Reply-To': reply_to},
+        cc=other_recipients
+    )
+
+    if uploaded_file:
+        filename = uploaded_file.name
+        content = uploaded_file.read()
+        email.attach(filename=filename, content=content)
+
+    # only fake the from email if it's an @dimagi.com account
+    is_icds_env = settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS
```

Comment:
nit: this is redundant now and can be removed since the ICDS environment does not exist anymore. Please feel to leave if don't want to make this change in this PR since its unrelated

Reply:
I think given that I simply moved what was being done for the SaaS bug report, I'll opt to rather leave this as is for now.